network-api/networkapi/fellows/templates/fellows_apply.html

```diff
@@ -43,8 +43,7 @@ <h3 class="h5-black mb-1">Tech policy professionals</h3>
         </div>
 
         <div class="col-12 offset-md-3 mt-2 mb-5">
-          <a href="https://mozilla.fluxx.io/user_sessions/new" class="btn btn-normal">apply to be a fellow today</a>
-          <p class="small-gray mt-2">The call closes <strong>Friday, April 20, 2018 at 11:59PM PST</strong>.</p>
+          <a href="#" class="btn btn-normal">applications will open soon</a>
```

Comment:
Can you add the `disabled` class so that the button doesn't appear clickable?

Reply:
good idea - done!